### ISSUE TITLE
New Chain/Discord

### ISSUE THREAD

**Issue by joeland1** [EmeraldMiningCo/Ebits#1]
What happened to discord?
ATM im trying to save chain, make it work. We know the issue for connecting.

**Comment by EmeraldMiningCo** [comment]
Autos going nuts or idk whats hes doing with the discord I just dont
understand how it came to this should of just been simple.

**Comment by joeland1** [comment]
Can you send me login credentials to the vultr account with funds? we need to create more vps to set up mn. dm me in discord, my id is in my aboutme.md

**Comment by joeland1** [comment]
chain is stable
https://bashupload.com/lDf7i/EbitsNewMN.zip  -   linux
https://mega.nz/#!TEUjACyD!SWlxmYsYXRoSwSwuR-bWLFAEdpwS3DjzbEWG2z5epqo  -  windows

**Comment by Epic1** [comment]
this guy is a scam artist he cant run a coin i have one up and running with mns and everything in less than 24 hours he is a  joke just trying to get free work from you, you will get nothing from him, he if he cant even setup seed nodes he need to give up lol
try buying a full coins instead of lying and trying to mode a free version lol you scamd enough money from people you should be able too

**Comment by Epic1** [comment]
also shortex has been wanred of your scam

**Comment by Epic1** [comment]
enjoy not getting listed anywere now

**Comment by Epic1** [comment]
you can keep deleting and i will keep posting everyone should know what a pile of shit you are and how you do nothing but steal from people

**Comment by EmeraldMiningCo** [comment]
Sounds good bro. You have no idea what your doing go ahead. Do what you
need. We habe things going and running. To be straight up ur just mad
because ur not part of the team. Somebody here cant control thier anger 👌

**Comment by Epic1** [comment]
good luck stopping the emails to the exchanges letting them know of your lies lo

**Comment by EmeraldMiningCo** [comment]
Do see yourself right now. Bro you personally have some mental issues you
need to chill. Scam wise sure. Whos money did I take? 0lz bro plz tell me?
Everyone from old discord is in new server things are good dont understand
ur issue. Swriously look at you.

**Comment by Epic1** [comment]
you have shit running your just stealing from people liek you have done for years 
your a low life pile of shit, annd trust your girl will leave you and everyone else will too your garbage and i have already talked to Shortex they ARE NOT LISTING YOU, they have not even talk to you, and now they wont as i showed then your decite already and want nothing toyou with you or EBITs enjoy

**Comment by Epic1** [comment]
i hate thieves and liar's and you are both

**Comment by Epic1** [comment]
my issue is you fliped out on me asking you for an update on stuff you have done 0 of what you said you would, you stole money from jsbond you stole money from zer0 on your pipe dreams, you just expect everyone to wait on you and that is fine, i pissed you wasted my time on your lies and the money i sent you in paypal lol think i forgot lol trust i will own ebits if you can ever get it launched now so many exchanges are not going to list you they dont like the chat logs proving your a scam lol i just hate liars and thieves i did nothing but support you and your bullshit and put faith in you when i knew you where just lying i will not very smart bro not at all, but your loss, not mine everyone will leave you they will get sick of you doing nothing but expect everyone to wait on you

**Comment by Epic1** [comment]
so enjoy back to telling people of your decit good luck bro your really gonna need it now

**Comment by EmeraldMiningCo** [comment]
Sounds good bro. Jbond2 is in chat and never mentioned to me that i stole
his money 👌 also that deal u made with zero is between u two so thats not
on me get ur facts straight.

**Comment by Epic1** [comment]
its cool i got my own coin and it works unlike yours do you need help lol fucking loser
